src/apis/downloadPdf.js

```diff
@@ -32,7 +53,12 @@ export default store => (includeAnnotations = true) => {
     return;
   }
 
-  downloadPdf(store.dispatch, {
-    includeAnnotations,
-  });
+  // legacy interface: instance.downloadPdf(true);
+  if (typeof options === 'boolean') {
```

Comment:
we can pass in a boolean only? i thought options had to be an object?

Reply:
Hmm we can pass in an object but this API also takes in a boolean argument for backwards compatibility.